extensions/github/src/components/PullRequestListItem.tsx

```diff
@@ -82,6 +83,13 @@ export default function PullRequestListItem({ pullRequest, viewer, mutateList }:
             icon={Icon.Sidebar}
             target={<PullRequestDetail initialPullRequest={pullRequest} viewer={viewer} mutateList={mutateList} />}
           />
+          {numberOfComments > 0 && (
+            <Action.Push
+              title="Show Comments"
+              icon={Icon.Message}
+              target={<PullRequestComments viewer={viewer} mutateList={mutateList} pullRequest={pullRequest} />}
+            />
+          )}
```

Comment:
This should be added below `Open in Browser` as it'll break people using ⌘ + ⏎ to open the PR in the browser.